queries/special.py

```diff
@@ -472,6 +487,18 @@ def _play_music(qs: str, q: Query) -> AnswerType:
     "hver er fallegust": {
         "answer": "Þú, kæri notandi, ert að sjálfsögðu fallegastur af öllum."
     },
+    "hver er fallegastur af öllum": {
```

Comment:
Consider replying with same gender as the question: "Hver er fallegust?" -> "Þú ert fallegust"?